gammapy/maps/region.py

```diff
@@ -185,17 +185,33 @@ def bin_volume(self):
 
         return bin_volume
 
-    def to_wcs_geom(self):
+    def to_wcs_geom(self, width_min=None):
         """Get the minimal equivalent geometry that
             contains the region.
 
+        Parameters
+         ----------
+            width_min : `~astropy.quantity.Quantity`
+            Minimal width for the resulting geometry.
+            Can be a single number or two, for 
+            different minimum widths in each spatial dimension.
+
         Returns
         -------
         wcs_geom : `~WcsGeom`
             A WCS geometry object.
         """
+        if width_min is not None:
```

Comment:
Does this work?
```python3
width = np.min([self.width.to_value("deg"), _check_width(width_min)], axis=0)
```
Where `_check_width()` is imported from `gammapy.maps.wcs`...

Reply:
oh yes, that is fantastic! once again, had no idea this existed, very useful function!